js/libs/build.js

```diff
@@ -271,6 +271,8 @@ Fliplet.Widget.instance('text', (widgetData) => {
         return $html;
       },
       saveChanges() {
+        this.cleanUpContent()
```

Comment:
You have plenty other places in the code that calls `debounceSave()` after calling `cleanUpContent()`. This suggests that if you make `cleanUpContent()` part of `saveChanges()`, then you should be able to delete many other instances of calling `cleanUpContent()`.

Reply:
@tonytlwu Removed the unnecessary ones.